components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -559,7 +596,13 @@ public OAuthAppDO[] getAppsForConsumerKey(String consumerKey)
                         authenticatedUser.setUserName(rSet.getString(USERNAME));
                         oauthApp.setApplicationName(rSet.getString(APP_NAME));
                         oauthApp.setOauthVersion(rSet.getString(OAUTH_VERSION));
+
```

Comment:
redundant blank line

Reply:
Added it to improve readability by separating the callback URL resolving logic.